src/common/util/src/os/win/win_shared_object_loader.cpp

```diff
@@ -70,9 +70,74 @@
 #endif
 
 #include <windows.h>
+#include <wintrust.h>
+#include <Softpub.h>
+
+// #include <wincrypt.h>
+// #include <wintrust.h>
+
+// #pragma comment (lib, "wintrust")
+
+namespace {
+bool verify_embedded_signature(LPCWSTR pwszSourceFile)
+{
+	GUID WintrustVerifyGuid = WINTRUST_ACTION_GENERIC_VERIFY_V2;
```

Comment:
Add links:
- https://github.com/konstantin89/windows-pe-signature-verifying/blob/master/pe-signature-utils/src/native_api_wrappers/TrustVerifyWrapper.cpp#L21
- https://learn.microsoft.com/en-us/windows/win32/seccrypto/example-c-program--verifying-the-signature-of-a-pe-file